tests/phpunit/tests/menu/wp-nav-menu.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/**
+ * @group menu
+ */
+class Test_Wp_Nav_Menu extends WP_UnitTestCase {
+
+	protected $menu_id        = 0;
```

Comment:
Should these be static?

Reply:
@costdev I think we can. Out of curiosity, why would it be better to use `static` here ?

Reply:
fixed in https://github.com/WordPress/wordpress-develop/pull/3357/commits/4188d743344ab940358c20b0b2a53be6506f91b6

Reply:
Hmm. Looks like converting `protected` to `static` generates unit test failures.

Reply:
@audrasjb Tests are failling because we still try to access the static properties using `$this`. We need to replace all instances of `$this->` to `self::$` in the test file for the static properties (`menu_id`,`lvl0_menu_item`,`lvl1_menu_item`,`lvl2_menu_item`)

* `$this->menu_id` should be rewrite on `self::$menu_id`,
* `$this->lvl0_menu_item` should be rewrite on `self::$lvl0_menu_item`,
* etc.

Tests should be all green after that !